ultet/src/main/scala/za/co/absa/ultet/dbitems/DBTable.scala

```diff
@@ -17,11 +17,11 @@
 package za.co.absa.ultet.dbitems
 
 import za.co.absa.ultet.dbitems.DBTableMember._
-import za.co.absa.ultet.model.{DatabaseName, SchemaName, UserName}
+import za.co.absa.ultet.model.{DatabaseName, SchemaName, UserName, TableName}
 
 //TODO checks on validity of entries
 case class DBTable(
-                   tableName: String,
+                   tableName: TableName,
```

Comment:
some of these changes, like this one, are already present in https://github.com/AbsaOSS/fa-db/pull/91/files -- but I think there should be no conflicts.

Reply:
Okay thanks for letting me know 👍